R/input-checks.R

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
should we add a wrapper that calls all of these and maybe returns a list of tibbles? And maybe prints a message that checks whether any of the tibbles have any rows and then either prints `"All stories, requirements, and tests match"` or `"You have {n} mismatches. Please check..."`?

Not sure if it's necessary, but it might be nice for a user just to be able call that one thing and see what they're missing.

Reply:
Sounds like a good idea.